client/data/marketplace/utils.ts

```diff
@@ -20,3 +21,18 @@ export const getPluginsListKey = (
 
 	return [ ...key, PLUGINS_CACHE_VERSION.toString(), ...keyParams ];
 };
+
+/**
+ * Get a single avatar URL from a given review.
+ * Tries to get a picture with the biggest resolution possible.
+ * @param {MarketplaceReviewResponse} review the review object
+ * @returns {string|undefined} the avatar URL when found
+ */
+export const getAvatarURL = ( review: MarketplaceReviewResponse ) => {
+	return (
+		review.author_avatar_urls[ '96' ] ??
+		review.author_avatar_urls[ '48' ] ??
+		review.author_avatar_urls[ '24' ] ??
+		undefined
```

Comment:
When none of those versions are found, a gray placeholder is shown as per today.
![CleanShot 2023-12-20 at 08 26 19@2x](https://github.com/Automattic/wp-calypso/assets/5039531/fb4225d0-cfa9-423d-af94-8fc077c027f4)

Do you know the backend response when the avatar is unavailable? I expected it to return the same fields but with a placeholder image, but it would be nice to confirm.

Reply:
The `author_avatar_urls ` object will always come in this structure, having the size as the key.

fbhepr%2Skers%2Sjcpbz%2Sjc%2Qvapyhqrf%2Serfg%2Qncv.cuc%3Se%3Qo2469qq6%231228%2Q1246-og

fbhepr%2Skers%2Sjcpbz%2Sova%2Sgrfgf%2Spber%2Sgrfgf%2Serfg%2Qncv%2Serfg%2Qpbzzragf%2Qpbagebyyre.cuc%3Se%3Q1sq0614n%231009%2Q1023-og

---

So it's safe to treat it this way, and there is still the gray icon as a guardrail in case anything goes wrong.